tests/test.rs

```diff
@@ -19,64 +19,68 @@ pub enum TestEnum {
 		a: i64,
 		#[revision(end = 3, convert_fn = "upgrade_three_b")]
 		b: f32,
-		#[revision(start = 3)]
+		#[revision(start = 3, default_fn = "default_three_c")]
 		c: f64,
-		#[revision(start = 3)]
+		#[revision(start = 3, default_fn = "default_three_d")]
 		d: String,
```

Comment:
If supported, shall we have a test where we remove a field from a tuple?

```Rust
ThreeTuple (
		#[revision(end = 3, convert_fn = "upgrade_three_1")]
		i32,
		#[revision(start = 3, default_fn = "default_three_2")]
		i64
)
```

Reply:
If you see the whole file you will notice there already is a test which does that:
```rust
#[revisioned(revision = 2)]
#[derive(Debug, Default, PartialEq)]
pub struct TestTuple2(
	#[revision(end = 2, convert_fn = "convert_tuple")] pub Vec<i64>,
	#[revision(start = 2)] pub Vec<f64>,
);
```

Reply:
That is supported I will add the test for it.

Reply:
That also immediately revealed a bug so good suggestion!